src/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -60,6 +60,47 @@ public OCFile createFromParcel(Parcel source) {
     public static final String PATH_SEPARATOR = "/";
     public static final String ROOT_PATH = PATH_SEPARATOR;
 
+    public enum AvailableOfflineStatus {
+
+        /**
+         * File is not available offline
+         */
+        NOT_AVAILABLE_OFFLINE(0),
+
+        /**
+         * File is available offline
+         */
+        AVAILABLE_OFFLINE(1),
+
+        /**
+         * File belongs to an available offline folder
+         */
+        AVAILABLE_OFFLINE_PARENT(2);
+
+        private final int value;
+
+        AvailableOfflineStatus(int value) {
+            this.value = value;
+        }
+
+        public int getValue() {
+            return value;
+        }
+
+        public static AvailableOfflineStatus fromValue(int value) {
```

Comment:
Nah,  I don't see disadvantages on having more attributes, I just meant that the benefit would be that you'd make sure that the same index is not used in 2 elements by mistake as it is assigned automatically